pkg/token/service.go

```diff
@@ -190,7 +190,9 @@ func (ts *tokenService) Start(ctx context.Context) error {
 			log.Info("Stopped token provider server")
 		}()
 
-		if err := daemon.WaitForServerReady(ts.tokenServer.Addr, ts.tokenServer.TLSConfig != nil); err != nil {
+		log.Warnln("🚨🚨🚨 Sleep 1s to trigger race condition of TLSConfig... 🚨🚨🚨")
+		time.Sleep(time.Second)
```

Comment:
delete later?

Reply:
reverted in [64c68cd](https://github.com/AthenZ/k8s-athenz-sia/pull/173/commits/64c68cde1ae98362603dcedf9b50c306a50ca424)